library/build.gradle

```diff
@@ -22,15 +23,22 @@ android {
     exclude 'LICENSE.txt'
     exclude 'META-INF/LICENSE.txt'
   }
+  sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
+  }
 }
 
 dependencies {
   compile 'io.reactivex:rxjava:1.1.8'
   compile 'io.reactivex:rxandroid:1.2.1'
+  compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 }
 
 task wrapper(type: Wrapper) {
   gradleVersion = '2.2.1'
 }
 
-apply from: '../maven_push.gradle'
\ No newline at end of file
+apply from: '../maven_push.gradle'
+repositories {
```

Comment:
It shouldn't be here.  Such things should be added to top-level build.gradle script.